### PR TITLE
Update RHEL base image builds to enable BYOL builds

### PR DESCRIPTION
**Pull request by zmarano**

### CODE REVIEW

File: daisy_workflows/image_build/enterprise_linux/build_installer.py [hunk 1]
```diff
@@ -32,6 +33,10 @@ def main():
   # Get Parameters
   repo = utils.GetMetadataParam('google_cloud_repo', raise_on_not_found=True)
   release = utils.GetMetadataParam('el_release', raise_on_not_found=True)
+  byol = utils.GetMetadataParam('rhel_byol', raise_on_not_found=False)
+  byol = True if byol == 'true' else False
```
Comment (crunk1) [inline daisy_workflows/image_build/enterprise_linux/build_installer.py]: byol = byol == 'true'

File: daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg [hunk 2]
```diff
@@ -3,31 +3,15 @@
...
-%post --nochroot
-# copy google supplied RPM packages to image.
-
-mkdir /mnt/sysimage/tmp/gce-rpm-install
-isodir=/mnt/isodir
-cp $isodir/*.rpm /mnt/sysimage/tmp/gce-rpm-install
-
-%end
-
 # 2
```
Comment (crunk1) [inline daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg]: Feel like renumbering these?

File: daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg [hunk 3]
```diff
@@ -3,31 +3,15 @@
...
-pkgs="google-rhui-client"
-for package in ${pkgs}; do
-    rpm="/tmp/gce-rpm-install/${package}.rpm"
-    [ -f ${rpm} ] || continue
-    yum -y install ${rpm}
-    KEYS="$(rpm -qlp ${rpm} | grep ^/etc/pki/rpm-gpg/)"
-    rpm --import ${KEYS}
-done
-rm -rf /tmp/gce-rpm-install
+yum -y install google-rhui-client-rhel6
```
Comment (crunk1) [inline daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg]: Me gusta.

Comment (crunk1) [reply daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg]: Also, is this an issue? I thought there were issues with exposing the Google RHUI clients publicly? Is the yum repo public?

File: daisy_workflows/image_build/enterprise_linux/build_installer.py (hunk 1 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/build_installer.py]: Done.

File: daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg (hunk 3 above)
Comment (zmarano) [reply daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg]: Its fine for now.

**Pull request merged**